src/Dispatcher/AlertDispatcher.php

```diff
@@ -9,6 +9,7 @@
 use Kevincobain2000\LaravelAlertNotifications\MicrosoftTeams\Teams;
 use Kevincobain2000\LaravelAlertNotifications\Slack\ExceptionOccurredPayload;
 use Kevincobain2000\LaravelAlertNotifications\Slack\Slack;
+use Throwable;
```

Comment:
if this is just used in `@param` for the phpdoc, it is better to remove unwanted use or change the phpdoc settings